src/content/app/custom-download/state/customDownloadReducer.ts

```diff
@@ -61,7 +60,7 @@ export function customDownload(
 
 function result(
   state: ResultState = defaultResultState,
-  action: ActionType<RootAction>
+  action: ActionType<any>
```

Comment:
We will at some point migrate custom download to redux-toolkit as well, I guess (if we even expect to keep developing it); but in the meantime, shouldn't we use `typeof customDownloadActions` rather than `any` here? See `src/objects` for where `customDownloadActions` were imported from.

Reply:
Changed it to `typeof customDownloadActions`